vision_evaluation/evaluators.py

```diff
@@ -1100,3 +1100,46 @@ def add_predictions(self, predictions, ground_truths):
 
     def get_report(self, **kwargs):
         return {self._get_id(): {group: eval.get_report(**kwargs) for group, eval in self._evaluators.items()}}
+
+
+class MeanLpErrorEvaluator(Evaluator):
+    """
+    Error evaluator for regression targets
+    Parameters:
+        p: Lp order of the norm used
+    """
+
+    def __init__(self, p: int):
+        assert p != 0
```

Comment:
P > 0?

Reply:
Good point - I was originally thinking p < 0 is ok, but I don't see where we'd use that. Changed.